cellforest/utils/r/run_r_script.py

```diff
@@ -13,6 +13,18 @@ def run_process_r_script(branch: "CellBranch", r_script_filepath: str, arg_list:
     )
 
 
+def run_plot_r_script(branch: "CellBranch", r_script_filepath: str, arg_list: list, process_name: str):
```

Comment:
We'll have to change this arg from `process_name` to `logfile_prefix` as well. (my bad)

Reply:
Ah, this may not be true, as I just noticed made a custom shell command for plotting. Not sure, though, as I don't fully understand the logfile naming in there